### DISCUSSION TITLE
Image Error

### DISCUSSION THREAD

**Discussion by beatricehc** [IBEXImagingCommunity/ibex_imaging_knowledge_base#213] [Q&A]
Hello!
I wanted to share this image and see if anyone knows why this error occurred in my image. This is a mouse lung imaged according to Ce3D-IBEX protocol, and this issue was not observed on the microscope and appeared when I opened the image in Imaris. It is likely a data transfer issue, but I wanted more information on how to prevent this. Thank you for your help!

![unnamed](https://github.com/user-attachments/assets/d835f3b7-c0a4-4d8a-9e75-3a4bf24b36ef)

**Comment by zivy** [comment]
Hello @beatricehc,

Possibly this problem is related to [this one](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/discussions/199)? Try the solution suggested there, "copy the original image in whatever format it was acquired to a local drive (not external), import into imaris and save to local drive".

If that doesn't resolve the problem, try viewing the original image using some other software (not the microscope's or imaris). This will help determine if the artifact already exists in the original data but the microscope's software is set up so that it isn't visible. If the artifact isn't in the original data, export it from this third software and then open in imaris.
